lib/runtimes/ServerlessBaseRuntime.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
```

Comment:
Sorry to nitpick, but can you rename these to `ServerlessRuntime[Type]` so that they are grouped together in our lib folder?